content/en/docs/virtualization.md

```diff
@@ -334,3 +334,65 @@ windows.2k22 | Microsoft Windows Server 2022
 windows.2k22.virtio | Microsoft Windows Server 2022 (virtio)
 windows.2k25 | Microsoft Windows Server 2025
 windows.2k25.virtio | Microsoft Windows Server 2025 (virtio)
+
+
+## Virtual Routers
+
+Starting with version [v0.27.0](https://github.com/cozystack/cozystack/releases/tag/v0.27.0),
+Cozystack can deploy virtual routers (also known as "router appliances" or "middlebox appliances").
+This feature allows you to create a virtual router based on a virtual machine instance.
+The virtual router can route traffic between different networks.
+
+## Creating a Virtual Router
+
+Creating a virtual router requires a Cozystack administrator account.
+
+1.  **Create a VM Instance**<br/>
```

Comment:
Explicit `<br/>` is better than implicit double trailing space. Must check that Hugo builds it correctly.